packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/stop-loss.jsx

```diff
@@ -60,24 +64,28 @@ const StopLoss = ({
 StopLoss.propTypes = {
     amount: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
     currency: PropTypes.string,
+    current_focus: PropTypes.string,
     has_stop_loss: PropTypes.bool,
     is_single_currency: PropTypes.bool,
     onChange: PropTypes.func,
     onChangeMultiple: PropTypes.func,
+    setCurrentFocus: PropTypes.func,
     stop_loss: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
     validation_errors: PropTypes.object,
 };
 
 export default connect(({ modules, client, ui }, props) => ({
     addToast: ui.addToast,
-    removeToast: ui.removeToast,
-    is_single_currency: client.is_single_currency,
     amount: modules.trade.amount,
     currency: modules.trade.currency,
+    current_focus: ui.current_focus,
```

Comment:
can we use `document.activeElement` to get the focused element instead of tracking it with a separate prop?

Reply:
Ah, yes 👍